tests/functional/spec/components/dropdown.spec.js

```diff
@@ -0,0 +1,138 @@
+const helpers = require('../../helpers');
+
+describe('Component: Dropdown', function() {
+  //Mandatory
+  describe('Given I start a Mandatory Dropdown survey', function() {
+
+    const DropdownMandatoryPage = require('../../pages/components/dropdown/mandatory/dropdown-mandatory.page');
+    const DropdownSummaryPage = require('../../pages/components/dropdown/mandatory/summary.page');
+    const schema = 'test_dropdown_mandatory.json';
+
+    it('When I have selected a dropdown option, Then the selected option should be displayed in the summary', function() {
+      return helpers.openQuestionnaire(schema)
+        .then(() => {
+          return browser
+            .selectByValue(DropdownMandatoryPage.answer(), "Rugby is better!")
+            .click(DropdownMandatoryPage.submit())
+            .getText(DropdownSummaryPage.dropdownMandatoryAnswer()).should.eventually.contain("Rugby is better!");
+        });
+    });
+
+    it('When I have not selected a dropdown option and click Continue, Then the default error message should be displayed', function() {
+      return helpers.openQuestionnaire(schema)
+        .then(() => {
+          return browser
+            .click(DropdownMandatoryPage.submit())
+            .getText(DropdownMandatoryPage.errorNumber(1)).should.eventually.contain("Select an answer to continue.");
+        });
+    });
+
+    it('When I have selected a dropdown option and I try select a default (disabled) dropdown option, Then the already selected option should be displayed in summary!', function() {
```

Comment:
- "try select" should be "try to select"
- To be consistent with the other statements this should end with `in the summary` (also dropping the !)